File: array/26.py
class Solution(object):
    def removeDuplicates(self, nums):
        i=0
        if len(nums)<1:
            return i
        for j,num in enumerate(nums):
            if num!=nums[i]:
                i+=1
                nums[i]=num
        return i+1
    
def main():
    c=Solution()
    test=[1,1,2]
    print(c.removeDuplicates(test))
    print(test)

if __name__ == '__main__':
    main()

File: array/26.js
var removeDuplicates = function(nums) {
  let i=0;
  if(nums.length<1) {
    return i;
  }
  for(let j=0;j<nums.length;j++){
    if (nums[j]>nums[i]) {
      i++;
      nums[i]=nums[j]
    }
  }
  return i+1
};

let test=[1,1,2];
console.log(removeDuplicates(test));